### ISSUE TITLE
2.1.1.2. Dat Ecosystem Survey

### ISSUE THREAD

**Issue by serapath** [dat-ecosystem/organization#16]
#8

## `@todo`
- [ ] make a survey on Google forms (or any other platform easy to use and to analyse data)
  - `@input` :package: [`old dat foundation twitter account`](https://twitter.com/dat_project) from #14
  - `@input` :package: [`chat message notification`](https://gitter.im/datproject/discussions?at=61939e72cd4972068baca502) from #14
  - `@input` :package: [`dat-ecosystem blog`](https://github.com/dat-ecosystem/blog) from #9
  - `@input` :package: [`dat-ecosystem blog github repo`](https://blog.dat-ecosystem.org/) from #9
  - `@input` :package: [`dat-ecosystem npm org`](https://www.npmjs.com/org/dat-ecosystem) from #11
  - `@input` :package: [`dat-ecosystem-archive repos`](https://github.com/dat-ecosystem-archive) from #11
  - `@input` :package: [`list of repos & maintainers`](https://hackmd.io/8uiDThl7S3OB-EC3JFUoqg?both#List-of-DAT-repositories-and-modules) #from 10
  * [ ] find projects, people, organizations related to hypercore protocol and the ecosystem
  * [ ] developers, projects (modules, companies..)
  * [ ] financial supporters (grant programs, patrons...)
  * [ ] media (tech bloggers, community newsletters
  - [ ] post survey periodically to get most results (at least 1 time per week for 1 month) on: irc+old gitter, hyper discord, dat discord, twitter, mailing list, friendly communities, ask people to share
  - [ ] analyse and make a list with: name, contact (email (optional, for future newsletters and dat ecosystem communication)), website, logo/avatar/profile pic, about/professional bio (related to p2p/hypercore + links), github (links to their p2p/hypercore related repos), location, language, yrs in the ecosystem, each entry tagged with the right category/ies
    - `@output` :question: `audience list from survey`
      - next #33
---

## `@info`
**Goals:**
find out who are the people interested in hypercore protocol, what are they building, what languages do they speak, what coding languages do they use, how old are they, why are they interested in this space, what are other things they care about...
We'd like to use this to reconnect with the community who is maybe not so active anymore, but has been interested in dat in the past years. Survey can also be used to share some important infos about dat-ecosystem with everyone who participates
maybe implicitely just through the way we phrase and structure the survey.

## estimated duration:
1.5 days

## estimated budget:
480 USD

**Comment by ninabreznik** [comment]
@deka can you add a link to the JS survey for inspiration

**Comment by aeriform-io** [comment]
on it.

**Comment by dpaez** [comment]
> @deka can you add a link to the JS survey for inspiration

Sure, [State of JS](https://stateofjs.com/) is definitively a great example of a survey directed to developers.

**Comment by ninabreznik** [comment]
> on it.

Wonderful. We're happy to have you on it :)

Let's do it then.  We are here to answer any questions you might have along the way :)

**Comment by aeriform-io** [comment]
I've done a first pass of questions—let me know if I'm on the right track !

—
We would like to reconnect with you, and so there are some things we would like to ask:

:: What is your name?
:: What is your age?
:: What is your chosen spoken language?
:: What programming language is your focus?
:: Can you tell us about what you are building?
:: What makes you interested in the Dat Ecosystem space?
:: How could the Dat Ecosystem be useful to you?
:: What would you like to see from the Dat Ecosystem?
:: Is there a particular aspect of the Dat Ecosystem that interests you?
:: Have you worked with the Hypercore Protocol or related technology before, if so can you tell us about it?
:: Is there something you would like to specifically contribute to the Dat Ecosystem?
:: Do you have a Github/Gitlab/Sourcehut repo you would like to share with us?
:: Is there an email address we can use to contact you directly? (This will not be shared with third-parties)
:: Would you like to join the mailing list for our newsletter?
:: Do you have any questions you would like to ask us?

**Comment by dpaez** [comment]
Hi @aeriform-io, good progress, I think we are on the right track!

Some things that comes to my mind at first. 
- I think `name` is not needed
- `Can you tell us about what you are building?` => looks quite open, maybe we can also ask about which industry their are working on, providing some options. We can also consider asking if they have some project related to dat/hyper. **Also ask more about working experience, current job, is it related with decentralization?, company name**
- `What makes you interested in the Dat Ecosystem space?` => right after/before we can ask something around how do they see or perceive dat ecosystem.
- `Is there a particular aspect of the Dat Ecosystem that interests you?` => maybe this questions should have some options with an `other` field 
- add some questions like: 
  - do you know about `dat consortium`? if no => provide link to consortium manifesto or some related resource
  - would your company/project consider join the `dat consortium`?
 
@ninabreznik do you have any other suggestions here?

**Comment by aeriform-io** [comment]
Okei I'll start to put this together in Google Forms and see what it needs to be added. I was considering Typeform but the trial only allows 10 responses, which isn't very useful !

**Comment by ninabreznik** [comment]
You can also check at Wufoo forms, Survey Monkey or Google forms. I used
these in the past, but I'm not sure about their model atm.

**Comment by dpaez** [comment]
Hi @aeriform-io, bringing this back again, let me know if you need any help or assistance! :+1:

**Comment by ninabreznik** [comment]
@dpaez @aeriform-io I assume this task can be assigned to someone else, so I will start looking for a person to execute it. We need the results soon as we are starting to work on the webpage.

**Comment by aeriform-io** [comment]
I can do it this week but if you want to assign it to someone else up to you. I was looking for an alternative to Google Forms, I did find one but hadn't been able to try it yet.

**Comment by ninabreznik** [comment]
> I can do it this week but if you want to assign it to someone else up to you. I was looking for an alternative to Google Forms, I did find one but hadn't been able to try it yet.

Hey, if you think you can finish it by the end of the week, then please let's do it, but if you don't manage by Sunday evening, I will really have to find someone else so we can push this forward.

I hope you understand that this task involves the survey itself and then the distribution of it and gathering the results and creating a 'report'. So, if you finish the survey by the end of the week, we will then need maybe 10 days to have the survey out and gather the responses and then to get the report done. Do you think you have the capacity to do all this in the next 3 weeks?

So, should we say
- end of this week survey questions
- then publishing the survey to get as many responses as possible (aiming for at least 50) (by June 1st)
- analysis and final report (by June 10th)

PS: I would recommend publishing the survey on any of the [free typeform alternatives](https://marketsplash.com/typeform-alternatives/) and alternatively maybe also on the Discord itself using a bot such as https://top.gg/bot/823271140410916986 - but you decide, maybe one form is enough and we will manage to get enough results). 

For posting on Twitter, just give me a little post and I can tweet with Dat ecosystem account :)

Please let me know what you think about this plan. And no hard feelings if you are currently busy, we just need to know so the task gets done on time and with some love <3

**Comment by aeriform-io** [comment]
Hi everyone, I have built a form in Tally. I like this platform but will need to see how it works out.

https://tally.so/r/w2EXGD I have made all of the questions optional so you can click through the pages but can set any to required later.

**Comment by serapath** [comment]
LGTM 🙂

**Comment by ninabreznik** [comment]
> Hi everyone, I have built a form in Tally. I like this platform but will need to see how it works out.
> 
> https://tally.so/r/w2EXGD I have made all of the questions optional so you can click through the pages but can set any to required later.

Hey, this is a pleasant surprise. I like how you structured it. It is open enough to probably cover all the edge cases. One question though, where will the responses be stored (so we can in the future revisit them if needed). Maybe you can invite us as co-editors or if it's possible export the full responses.

Here's the link to the Consoritum manifesto you can add to the form https://github.com/dat-ecosystem/organization/blob/main/MANIFESTO.md

And maybe at the end of the form we could also add the important links:
- webpage: (working on the new page atm) https://dat-ecosystem.org/
- discord: https://discord.gg/9rqEF6WVnJ
- twitter: https://twitter.com/dat_ecosystem
- github: https://github.com/dat-ecosystem

Otherwise, I'd say, let's go with this and let's share it with the people. I think weekends are great for this, since people have more time. Not sure just how to ask the community for the response so that they actually go and give us their 10min to fill out everything.

**Comment by aeriform-io** [comment]
It appears all of the answers are in the Tally platform and looks good for what it collects and can export a CSV. I can invite team members but that is a paid upgrade we can do that later if needs be ?

There's some clean up I need to do which I am working on now, and I'll add the links thank you Nina.

**Comment by aeriform-io** [comment]
Okei I have updated it, are there any extra things such as programming languages that should be added, and any fields you think should specifically be required ?

**Comment by ninabreznik** [comment]
> It appears all of the answers are in the Tally platform and looks good for what it collects and can export a CSV. I can invite team members but that is a paid upgrade we can do that later if needs be ?
> 
> There's some clean up I need to do which I am working on now, and I'll add the links thank you Nina.

Yeah, I think csv will be fine.

**Comment by ninabreznik** [comment]
> Okei I have updated it, are there any extra things such as programming languages that should be added, and any fields you think should specifically be required ?

Maybe we could ask if there is also another p2p/blockchain technology they are familiar with and which is that. And also, would it make sense to ask for the country of residence? Other than that i think you've covered things pretty good.

@dpaez do you have any thoughts?

**Comment by aeriform-io** [comment]
> > Okei I have updated it, are there any extra things such as programming languages that should be added, and any fields you think should specifically be required ?
> 
> Maybe we could ask if there is also another p2p/blockchain technology they are familiar with and which is that. And also, would it make sense to ask for the country of residence? Other than that i think you've covered things pretty good.
>

Added these questions

**Comment by dpaez** [comment]
Thank you both @aeriform-io and @ninabreznik for working on and reviewing this! ⭐ 

@aeriform-io everything looks good. As nina mentioned, it is important to have a clear way to access the results. If you can share access with some of the consortium members (like nina or myself) that would be great.

Regarding the survey, I have just some a couple of details to share. I think we may rephrase this sentence: 

`Is there an email address we can use to contact you directly?` Adding some information about **why** we need it. Like, this can be used for future communications, sharing events and things like that. It won't be frequent.

Then, in the next question, when we ask if they want to join our newsletter, we should set that as radio group with `yes` and `no`. And they can choose only one, of course. 

That's my review. :) As soon it is ready we will be good to start sharing this. We would need to add some context, like we are in the process of a relaunch and we want to connect with our audience again...

**Comment by ninabreznik** [comment]
I did a draft of the copy for the website, so maybe you can use stuff from here to make a nice and inviiting post when sharing the survey (or should we find a catchy synonym that makes it sound more exciting - like 'a little quiz' (to make it sound cute and easy peasy) or a 'self exploration journey' (if we want to be funny), I don't know, you're the native speaker, you can maybe come up with something cool)
https://hackmd.io/@T6Wf5EsOQKe-6wyPjJPtuw/BySbidHDc

**Comment by ninabreznik** [comment]
One last thing - what if we replace the question mentioning the consortium (if you know consoritum and if you are thinking about joining) with are you thinking of joining the dat ecosystem growth program. Because we are actually planning to guide people through different phases. To first join the growth program and get help/mentorship working with hyper. Then maybe some micro grant for supporting the project and only after a while projects that are serious and have shown their really care about the ecosystem and the p2p space, they might be invited to join the consortium. So, it is not really ment as the first step and can maybe be confusin if we ask them about it this way.

Maybe we should therefore replace the whole consoritum thing (2 questions if I am not wrong) with Dat ecosystem Growth program - do they know it and would they consider joining (and you can link to this Github copy instead of the Manifesto that will not say much to the random ecosystem person)

**Comment by ninabreznik** [comment]
Updated the link https://hackmd.io/@T6Wf5EsOQKe-6wyPjJPtuw/BySbidHDc

**Comment by dpaez** [comment]
> One last thing - what if we replace the question mentioning the consortium (if you know consoritum and if you are thinking about joining) with are you thinking of joining the dat ecosystem growth program

Agree. So the final question will be something like this:

```md
Are you thinking of joining the dat ecosystem growth program? (Dat Ecosystem Growth Program is a new program aimed to help new projects. You can read more about it [here](...))
```

What do you think @ninabreznik ?

**Comment by ninabreznik** [comment]
> > One last thing - what if we replace the question mentioning the consortium (if you know consoritum and if you are thinking about joining) with are you thinking of joining the dat ecosystem growth program
> 
> Agree. So the final question will be something like this:
> 
> ```lisp
> Are you thinking of joining the dat ecosystem growth program? (Dat Ecosystem Growth Program is a new program aimed to help new projects. You can read more about it [here](...))
> ```
> 
> What do you think @ninabreznik ?

I was thinking to replace this last question with the Dat Ecosystem + the link below
![Screenshot_2022-05-20_21-50-18](https://user-images.githubusercontent.com/1076427/169608859-0b166ac3-89ba-41f5-8b1c-a3a24ffb809b.png)

**Comment by aeriform-io** [comment]
I have updated based on this feedback, but I'm not seeing a link to information about the growth program, only the form to join it ?

**Comment by ninabreznik** [comment]
> I have updated based on this feedback, but I'm not seeing a link to information about the growth program, only the form to join it ?

Here you go and thanks for the updates https://hackmd.io/@T6Wf5EsOQKe-6wyPjJPtuw/BySbidHDc

**Comment by aeriform-io** [comment]
oh it's literally just this ?

**Comment by ninabreznik** [comment]
> oh it's literally just this ?

Yes, sadly. We are working on the webpage, but it's a chicken and egg problem - need to get some info about the community to be able to then make a page and explain the whole program there :)

**Comment by aeriform-io** [comment]
Of course, I've updated it with this for now

**Comment by aeriform-io** [comment]
but I think we can go ahead with this if you guys are happy with it. I made a little graphic to go in the tweet if you like.

I also thought about the phrasing of it, maybe something like:

"We want to build a garden on top of the Hypercore P2P Protocol, with you. If you have a moment to answer some questions it would really help us!"

![des_cover](https://user-images.githubusercontent.com/15712187/169651051-36019002-05af-4fe0-8051-10102669f62d.png)

**Comment by ninabreznik** [comment]
I like the text. +1 from me. Sounds kind and inviting. The only thing is
the logo, maybe for now we should use the old elements (until we have the
new page and we figure out how exactly do we want to look and feel). But it
definitely looks good, just maybe not familiar, so fir now familiarity is
the key, imho

**Comment by serapath** [comment]
heya,
thx for all the work already.
first impression when i saw the notification was wow :-) ...but then i quickly remembered that we are still in the process of find8ng the new visual identity or brand and the graphic is neat, but also quite opinionated. the different website task includes going through this.

...but more than that, we did not reach out to the broader dat community for a long time, so i strongly assume any visuals should remind people of what they were familiar back then.

as interesting as your graphic looks, the dat community which hasnt heard any life sign from dat in a long time are probably most familiar with https://dat-ecosystem.org/datproject.org


or alternatively going all the way back to the roots https://dat-ecosystem.org/dat-data.com/ ...which interestingly included a bitcoin address back then already :-)

or this one, which was a later iteration https://dat-ecosystem-archive.github.io/old-website/


so white and bright green is where dat came from. later it included the new colors, but i pwrsonally wouldnt mind including some of the earlier styles to guide us 🙂

**Comment by aeriform-io** [comment]
fair, it was only meant for the survey, I'll refer back to earlier graphics

**Comment by aeriform-io** [comment]
![des_cover](https://user-images.githubusercontent.com/15712187/169657176-68008412-6a4b-47c1-9115-b2f8be9637b7.png)

how be this

**Comment by ninabreznik** [comment]
Yay, i like it way more and it is so much more familiar 😍

**Comment by aeriform-io** [comment]
I think all good to go if someone wants to post, I assume from the official Dat twitter ? I can post myself in various places too if needs be

**Comment by ninabreznik** [comment]
> I think all good to go if someone wants to post, I assume from the official Dat twitter ? I can post myself in various places too if needs be

Yay, I started sharing the survey so let's hope for some good amount of responses. We will have to share this multiple times, to be able to catch people in the right time (when they are online and have time).

Would you maybe share it in the Hypercore discord? I shared on Twitter and in Dat ecosystem discord. Will also go and share it in the old Gitter channel.

Let's all help spread this! @dpaez @serapath

**Comment by dpaez** [comment]
This is great! Already tweet about it. Thanks for your effort! @ninabreznik @aeriform-io 👏

**Comment by ninabreznik** [comment]
@aeriform-io how is the the survey going? Did we get any responses? Just to see and make a plan how to push the it out more to get more responses...

**Comment by aeriform-io** [comment]
we are currently at 28 submissions

**Comment by ninabreznik** [comment]
I just tweeted the last call for the survey and then I think at the end of the week we can wrap it up and export the results and make a little summary/report of most important findings. After you do this, @aeriform-io you can open an expense on Open Collective so we can also pay for your work :)

**Comment by ninabreznik** [comment]
So, re the report. The goal would be to quickly understand the core characteristics of the audience. Basically a summary for each question and if there is a significant correlation between certain answers also to point that out (i.e. those who say they have built on hypercore before are also most likely to...). Maybe in a format that could be shared with the community itself.

But to do an actual deep analysis we would need some serious programs and hypothesis and we don't have the budget for that and it is also not needed, we are not some crazy corporation trying to microanalyse their users.

Does this make sense?
Also, the csv file would be really nice to have somewhere stored, maybe we'll make a private repo in order to make the responses public to anyone so would you export the file and sent us a workhole link perhaps?

**Comment by aeriform-io** [comment]
I've put together a first pass of the report, is there anything else specific it should have ?
https://docs.google.com/document/d/1P6UptLWJowd822HkNqI_p6Hn2x86NnHZtnGs8QHllSA

I can provide all of the responses, but I feel like I should share that privately with Nina ?

**Comment by ninabreznik** [comment]
THanks a lot @aeriform-io this looks great. Could you please send me the csv file with all the responses to my email (so I can store it and make it available for other consoritum members - this we sadly can't publish openly due to GDPR).

I would kindly ask for 2 more things - I will make a folder under dat-ecosystem organization and I'd ask you to make a PR to add 2 files: one would be the report and the other just emptu questions (if we decide to repeat the survey next year).

And then you can go to https://opencollective.com/dat and click 'Submit expense' and fill out the 'invoice' with agreed budget (on top of this page).

**Comment by ninabreznik** [comment]
Here's the link to the organization repo https://github.com/dat-ecosystem/organization/tree/main/communication

**Comment by ninabreznik** [comment]
@aeriform-io Your expense has been approved. I have to wait for 12h to see if there is any objection from other consoritum members before executing so you can most likely see is paid tomorrow.

![Screenshot_2022-06-18_20-42-15](https://user-images.githubusercontent.com/1076427/174454946-ba8bca14-9544-4ff1-9c7b-a742f8e74f5f.png)

**Comment by ninabreznik** [comment]
Hi,

I just realized I can't execute the payment myself. The invoice has been approved and it will be paid by our fiscial sponsor on Opne Collective (Dat ecosystem doesn't have a bank account) in the next days. If you are in a hurry, please let me know and I can ask them to speed the process.

**Comment by aeriform-io** [comment]
hi, something was assigned to me but I don't know what it is

**Comment by ninabreznik** [comment]
Don't worry, if so, then it is jaut so we can know who has been working on
it. Sorry about that 😊
